### PR TITLE
Adapt the debug and test scripts for `vscode-jest-runner` to be compatible with Vitest.

### PR DESCRIPTION
**Pull request by holgerkoser**
**What this PR does / why we need it**:
We recently transitioned our testing framework from Jest to Vitest. Following this change, we encountered issues with debugging our tests, as the existing configuration was no longer compatible. To resolve this, we modified the configuration of [`vscode-jest-runner`](https://marketplace.visualstudio.com/items?itemName=firsttris.vscode-jest-runner) and adapted the necessary scripts to ensure seamless integration with Vitest. These changes have restored our ability to debug tests effectively within Visual Studio Code.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix developer
Fixed debugging issues by updating vscode-jest-runner config and scripts.
```

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -41,8 +41,9 @@
     "**/coverage": true,
     "**/dist": true
   },
-  "jestrunner.enableYarnPnpSupport": true,
-  "jestrunner.jestCommand": "node scripts/jest.cjs",
-  "jestrunner.yarnPnpCommand": "../../scripts/yarn.cjs",
   "jestrunner.changeDirectoryToWorkspaceRoot": true,
+  "jestrunner.jestCommand": "node scripts/test.cjs",
+  "jestrunner.enableYarnPnpSupport": true,
+  // The command is prefixed with `.yarn/releases/`, so we need `../..` (see https://github.com/firsttris/vscode-jest-runner/blob/master/src/jestRunner.ts#L180)
```
Comment (petersutter) [inline .vscode/settings.json:47]: use permanent link
```suggestion
  // The command is prefixed with `.yarn/releases/`, so we need `../..` (see https://github.com/firsttris/vscode-jest-runner/blob/4dea33a0f692a8f1519dbb8b2567787ffe1843da/src/jestRunner.ts#L180)
```

File: scripts/debug.cjs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+//
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
```
Comment (petersutter) [inline scripts/debug.cjs:3]: new file should use latest year
```suggestion
// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

**Review by petersutter — APPROVED** [review]
minor findings, otherwise /lgtm

**Review by grolu — APPROVED** [review]
/lgtm

**Pull request merged**